### ISSUE TITLE
VersionBot: Nice to have: Block normal merges

### ISSUE THREAD

**Issue by pimterry** [resin-io-modules/resin-procbots#64]
[At least once](https://github.com/resin-io/resin-ui/pull/523) I've forgotten about versionbot, and merged PRs manually, leading to an annoying manual version update process. It'd be nice to versionbot took total responsibility of PRs (except for no-checks), and blocked normal merges completely, so that I can't accidentally usurp it and screw things up.

**Comment by lurch** [comment]
AFAIK there'd be no way for VersionBot to completely block normal merges, but at least on Etcher (which isn't using VersionBot yet) the repo is set up so that only the project-maintainer is allowed to merge PRs.

**Comment by pimterry** [comment]
Still allows things to go wrong if you're the maintainer :smile:. The UI is set up to block merges if there's a failing status check, and Versionist does fail and block merges there when there's no change-type footer, it's just that it then unblocks them once there is. If they simply stayed blocked that'd solve this. Might have some other UX implications though (might mean that status markers on commits are always ❌?)

**Comment by lurch** [comment]
OTOH, preventing any "manual override" of VersionBot by the project-maintainer if / when things go wrong, might cause bigger problems? \*shrug\*

**Comment by hedss** [comment]
Several issues have now been seen in reverse, where people have hit the 'Merge' button without thinking and got `master` out of sync. This is a problem, and one solution is to hide the button under another status check that only resolves when the PR is approved. We actually want to remove the `procbots/versionbot/ready-to-merge` label (see #72).

**Comment by hedss** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [VersionBot: Nice to have: Give feedback on the changelog lines to be added in advance](https://trello.com/c/oc7rzTDh/9-versionbot-nice-to-have-give-feedback-on-the-changelog-lines-to-be-added-in-advance)
